contract/gql/gql_types.py

```diff
@@ -17,9 +18,6 @@ class Meta:
             "id": ["exact"],
             "code": ["exact", "istartswith", "icontains", "iexact"],
             **prefix_filterset("policy_holder__", PolicyHolderGQLType._meta.filter_fields),
-            "amount_notified": ["exact", "lt", "lte", "gt", "gte"],
-            "amount_rectified": ["exact", "lt", "lte", "gt", "gte"],
-            "amount_due": ["exact", "lt", "lte", "gt", "gte"],
             "date_payment_due": ["exact", "lt", "lte", "gt", "gte"],
```

Comment:
cannot you leave them even if you don't plan to use them yet ?

Reply:
Ok. You are right. It should not affect to my changes with new filter "amountFrom/To"